subsys/net/l2/ethernet/gptp/gptp_data_set.h

```diff
@@ -244,6 +244,12 @@ struct gptp_global_ds {
 
 	/** A Grand Master is present in the domain. */
 	bool gm_present;
+
```

Comment:
Please put the structs after the other structs few lines above (trying to avoid extra holes here).

Reply:
If pahole does not say there are holes in this struct, then I am fine with current placement.